src/nitserial.nit

```diff
@@ -155,13 +157,47 @@ for mmodule in mmodules do
 		module_path += ".nit"
 	end
 
+	var target_modules = null
+	var importations = null
+	var mgroup = mmodule.mgroup
+	if toolcontext.opt_depth.value == 1 then
+		# group
+		if mgroup != null then target_modules = mgroup.mmodules
```

Comment:
Maybe you want do force the loading of all modules of the group by calling `ModelBuilder::visit_group`. before Without it, `MGroup::mmodules` contains only the modules loaded.